docs/topics/crud-mission-intro.adoc

```diff
@@ -14,6 +14,7 @@ The booster application exposes an HTTP API, which provides endpoints that allow
 * Execute an HTTP `DELETE` request on the `api/fruits/*` endpoint, passing in a valid ID as an argument. This removes the item with the specified ID from the database and returns an HTTP code `204` (No Content) as a response. If you pass in an invalid ID, the call results in an HTTP error `404`.
 
 ifndef::http-api-nodejs[]
+.Integration Tests
```

Comment:
Does the following paragraph (after `endif::http-api-nodejs[]`) also belong to "Integration Tests"?

Reply:
Actually I just need to remove the heading altogether, that restores this back to its initial state.

Reply:
The integration tests are not actually in this section so the title is not appropriate here.